client/web/src/components/CopyableText.tsx

```diff
@@ -64,9 +73,17 @@ export class CopyableText extends React.PureComponent<Props, State> {
                             variant="secondary"
                             aria-label="Copy"
                         >
-                            <Icon aria-hidden={true} svgPath={mdiContentCopy} /> {this.state.copied ? 'Copied' : 'Copy'}
+                            <Icon aria-hidden={true} svgPath={mdiContentCopy} />{' '}
+                            {this.props.secret ? '' : this.state.copied ? 'Copied' : 'Copy'}
                         </Button>
                     </div>
+                    {this.props.secret && (
+                        <div className="input-group-append">
+                            <Button onClick={this.onClickSecretButton} variant="secondary" aria-label="Secret">
```

Comment:
```suggestion
                            <Button onClick={this.onClickSecretButton} variant="secondary" aria-label={this.state.secretShown ? 'Reveal secret value as text' : 'Hide secret value'}>
```
maybe? Feels a bit more expressive to a person using a screen reader